sample-lnd.conf

```diff
@@ -864,49 +1036,56 @@ litecoin.node=ltcd
 ; amount of attempted channels will still respect the maxchannels param.
 ; autopilot.allocation=0.6
 
-; Heuristic to activate, and the weight to give it during scoring. (default:
-; top_centrality:1)
-; autopilot.heuristic=preferential:1
+; Heuristic to activate, and the weight to give it during scoring. 
+; Default:
+;   autopilot.heuristic={top_centrality:1}
+; Example:
+;   autopilot.heuristic=preferential:1
```

Comment:
We should probably format both the default value and example the same way. So to make it easier for the script update the example to `{preferential:1}`?